src/main/java/com/example/moviemanager/controller/ApiError.java

```diff
@@ -0,0 +1,40 @@
+package com.example.moviemanager.controller;
+
+import java.time.LocalDateTime;
+
+public class ApiError {
+
+    private int status;
```

Comment:
Maybe it makes more sense to have HttpStatus but not an int value of it. I think status is more readable in response than numeric value.